### PR TITLE
feat: 1585 Document signing

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
Screenshot 2020-06-23 at 16 29 19

### CODE REVIEW

File: cases/generated_documents/signing.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    BASE_DIR,
+    CERTIFICATE_PATH,
+    CERTIFICATE_PASSWORD,
+    SIGNING_REASON,
+    SIGNING_LOCATION,
+    SIGNING_EMAIL,
+)
+
+SIGNATURE_TITLE = "Digital Signature"
+FONT = os.path.join(BASE_DIR, "assets", "fonts", "Helvetica.ttf")
```
Comment (jfaracik) [inline cases/generated_documents/signing.py:21]: Are these documents going to be visible to exporters? If so could we use the GDS Transport font?

Comment (BenAndrew-Kainos) [reply cases/generated_documents/signing.py:21]: Sure

Comment (BenAndrew-Kainos) [reply cases/generated_documents/signing.py:23]: Screenshot 2020-06-25 at 14 10 42

File: .circleci/config.yml [hunk 2]
```diff
@@ -91,6 +95,10 @@ jobs:
       - restore_cache:
           key: dependencies-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
 
+      - run:
+          name: Install OS libraries
+          command: sudo apt-get install swig;
```
Comment (Mark-Spiers-kainos) [inline .circleci/config.yml:100]: don't think I have seen a semicolon in a command before

**Pull request merged**